src/Tizen.NUI.Components/Controls/Pagination.cs

```diff
@@ -185,6 +188,108 @@ public int IndicatorCount
             }
         }
 
+        private void OnIndicatorColorChanged(float r, float g, float b, float a)
+        {
+            IndicatorColor = new Color(r, g, b, a);
+        }
+
+        /// <summary>
+        /// Color of the indicator.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public Color IndicatorColor
+        {
+            get
+            {
+                return new Color(OnIndicatorColorChanged, indicatorColor);
+            }
+            set
+            {
+                if (value == null)
+                {
+                    return;
+                }
+
+                if (indicatorColor == null)
+                {
+                    indicatorColor = new Color((Color)value);
+                }
+                else
+                {
+                    if (indicatorColor == value)
+                    {
+                        return;
+                    }
+
+                    indicatorColor = value;
+                }
+
+                if (indicatorCount == 0)
+                {
+                    return;
+                }
+
+                for (int i = 0; i < indicatorCount; i++)
+                {
+                    if (i == selectedIndex)
+                    {
+                        continue;
+                    }
+
+                    ImageVisual indicator = indicatorList[i];
+                    indicator.MixColor = indicatorColor;
+                    indicator.Opacity = indicatorColor.A;
+                }
+            }
+        }
+
+        private void OnSelectedIndicatorColorChanged(float r, float g, float b, float a)
+        {
+            SelectedIndicatorColor = new Color(r, g, b, a);
+        }
+
+        /// <summary>
+        /// Color of the selected indicator.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public Color SelectedIndicatorColor
```

Comment:
yes... this property has the same problem...

Reply:
Thank you for your comment. It helps a lot!
By discussing with NUI members, the Setter of Color class properties is going to be deprecated.
i.e. R, G, B, A can be assigned in Color constructor.